1080i/Home.xml

```diff
@@ -214,7 +214,9 @@
                 <onfocus>ClearProperty(OptionsMenuFocused,Home)</onfocus>
                 <onfocus>SetProperty(TMDbHelper.WidgetContainer,301,Home)</onfocus>
 
-                <onup condition="!Skin.HasSetting(DisableSubmenuOnUp)">SetFocus(302,0,absolute)</onup>
+                <onup condition="Skin.String(HomeMenu.OnUp,Submenu)">SetFocus(302,0,absolute)</onup>
```

Comment:
Default setting should be submenu onup to retain current behaviour 
i.e. check for `String.IsEmpty(Skin.String(HomeMenu.OnUp))`

Reply:
Fixed to retain original default of opening submenu